bin/check_multi-line-string-array_contents

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# This script checks that multi-line-string-array only contains valid keys.
+
+failed=0
+
+check_keys() {
+    json_file=$1
+    echo "Checking 'multi-line-string-array' contents in $json_file"
+
+    present_keys=$(jq '[ .cases[] | recurse(.cases[]?) | ."multi-line-string-array" | select(. != null)[] ] | unique ' $json_file)
+
+    allowed_keys=$(jq '[ .cases[] | recurse(.cases[]?) | .input | keys[] ] | unique + ["expected"]' $json_file)
+
+    invalid_keys=$(jq --null-input \
+                      --argjson allowed "$allowed_keys" \
+                      --argjson present "$present_keys" \
+                      --raw-output '$present - $allowed | join("\n")')
+
+    if [ ! -z "$invalid_keys" ]; then
+        echo "Invalid multi-line-string-array entries:"
```

Comment:
As far as I can determine, there can be two sources of errors:

1. There's a line `"multi-line-string-array": ["foo"]`, but there is no `input` parameter named `"foo"`.
2. There's a line `"multi-line-string-array": ["foo"]`, but the `input` parameter named `"foo"` is not an array of strings.

#1547 tests 1, and the spec changes enforce 2 partially: is an array, but not necessarily of strings.

I would like if the CI script would say something more informative like:

```suggestion
        echo "The following multi-line-string-array input parameters did not occur in any tests:"
```